CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Applies new local tokens to `BannerText` (#470)
 - Update the Incentives component to handle CMS data (#474)
+- Fixes `ProductCard` bordered variant (#482)
```

Comment:
Can you move this entry to the beginning? [The latest version comes first.](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.)